help/en/html/doc/Technical/gitdeveloper.shtml

```diff
@@ -96,10 +99,59 @@
       working on that new branch, which will hold whatever you
       commit in later steps.</p>
 
-      <h3>Commit Changes to Your Local Repository</h3>You should
+<h3>Develop and Test Locally</h3>
+    Now that you have a copy of the code available, you can directly
+    work in it to develop any changes you want to make.
+    There are several different ways to do that, see the sidebar,
+    including 
+        <a href="Ant.shtml">using Ant from the command line</a>,
+        <a href="NetBeans.shtml">using the NetBeans IDE</a>, and 
+        <a href="Eclipse.shtml">using the Eclipse IDE</a>.
```

Comment:
You might also mention IntelliJ as an IDE for JMRI in this list. It has a its own help page ``IntelliJ.shtml``.

Reply:
> You might also mention IntelliJ as an IDE for JMRI in this list. It has a its own help page IntelliJ.shtml.

Thanks for catching that, done!